Library/Formula/jp2a.rb

```diff
@@ -1,19 +1,21 @@
-require 'formula'
-
 class Jp2a < Formula
-  homepage 'http://csl.sublevel3.org/jp2a/'
-  url 'https://downloads.sourceforge.net/project/jp2a/jp2a/1.0.6/jp2a-1.0.6.tar.gz'
-  sha1 '8d08a7f9428632c02351452067828af215afe2cf'
+  homepage "http://csl.sublevel3.org/jp2a/"
+  url "https://downloads.sourceforge.net/project/jp2a/jp2a/1.0.6/jp2a-1.0.6.tar.gz"
+  sha1 "8d08a7f9428632c02351452067828af215afe2cf"
+
+  option "without-check", "Skip compile-time tests."
 
-  depends_on 'jpeg'
+  depends_on "jpeg"
 
   def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "test" if build.with? "check"
+    system "make", "install"
   end
 
   test do
-    system "#{bin}/jp2a", "-V"
```

Comment:
@mikemcquaid yes it fails:

```
$ unset TERM
$ brew test jp2a
Testing jp2a
tput: No value for $TERM and no -T specified
==>

/usr/local/Cellar/jp2a/1.0.6/bin/jp2a
/usr/local/Library/Homebrew/test/fixtures/test.jpg

Environment variable TERM not set.
Error: jp2a: failed
Failed executing: /usr/local/Cellar/jp2a/1.0.6/bin/jp2a /usr/local/Library/Homebrew/test/fixtures/test.jpg
/usr/local/Library/Homebrew/formula.rb:703:in `block in system'
/usr/local/Library/Homebrew/formula.rb:660:in `open'
/usr/local/Library/Homebrew/formula.rb:660:in `system'
/usr/local/Library/Formula/jp2a.rb:26:in `block in <class:Jp2a>'
/usr/local/Library/Homebrew/formula.rb:613:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:21:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:610:in `run_test'
/usr/local/Library/Homebrew/cmd/test.rb:59:in `block (2 levels) in test'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/cmd/test.rb:58:in `block in test'
/usr/local/Library/Homebrew/cmd/test.rb:37:in `each'
/usr/local/Library/Homebrew/cmd/test.rb:37:in `test'
/usr/local/Library/brew.rb:135:in `<main>'
```

Note I also get messages about `TERM` when running a simple `brew info`:

```
$ brew info jp2a
jp2a: stable 1.0.6 (bottled)
http://csl.sublevel3.org/jp2a/
/usr/local/Cellar/jp2a/1.0.6 (8 files, 88K) *
  Built from source
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/jp2a.rb
tput: No value for $TERM and no -T specified
==>
Required: jpeg ✔
tput: No value for $TERM and no -T specified
==>
--without-check
	Skip compile-time tests.
```